Segmentation violation

$ docker run -v /home/mspasiano/git/sigla-main:/files italia/publiccode-parser-go

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x6fc2ff]

goroutine 1 [running]:
main.init.0()
	/go/src/github.com/italia/publiccode-parser-go/pcvalidate/pcvalidate.go:23 +0x7f

Hi

Thanks @mspasiano, same problem with last docker version:
```
$ docker run -v $(pwd):/files italia/publiccode-parser-go files/publiccode.yml
Unable to find image 'italia/publiccode-parser-go:latest' locally
latest: Pulling from italia/publiccode-parser-go
df20fa9351a1: Pull complete
7163173fb616: Pull complete
Digest: sha256:cf45e6d7c97fe4957389acdeb5a0383a55576244054909ef934b68e12ad759c0
Status: Downloaded newer image for italia/publiccode-parser-go:latest
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x6fc2ff]

goroutine 1 [running]:
main.init.0()
	/go/src/github.com/italia/publiccode-parser-go/pcvalidate/pcvalidate.go:23 +0x7f
```

--- a/pcvalidate/pcvalidate.go
+++ b/pcvalidate/pcvalidate.go
@@ -19,8 +19,10 @@ var (
 
 func init() {
 	if version == "" {
-		info, _ := debug.ReadBuildInfo()
-		version = info.Main.Version
+		version = "devel"
+		if info, ok := debug.ReadBuildInfo(); ok {
+			version = info.Main.Version
+		}
 	}
 	if date == "" {
 		date = "(latest)"

